### DISCUSSION TITLE
How to install DgnV8Converter?

### DISCUSSION THREAD

**Discussion by JohnMajerle** [iTwin/itwinjs-core#5405] [Q&A]
Recent versions of MicroStation CONNECT doesn't appear to install DgnV8Converter 2.0 like it used to.   Is it still supported?  If so, what it the current proper way to install it?

**Comment by kanademayuresh** [comment]
It is not support anymore.
We used to provide a functionality to convert i-model to iModel 1.5,1.6 and 2.0, using Publish functionality in MicroStation.
We have deprecated this functionality and hence DgnV8Converter 2.0 is not delivered.
Please take a look at iTwin Synchronizer and iTwin Snapshot tool to see if it solves your use case.

**Comment by JohnMajerle** [comment]
The iTwin SnapShot tool doesn't work for us because it doesn't provide a command line interface.
